SofaKernel/modules/SofaBaseMechanics/DiagonalMass.h

```diff
@@ -19,11 +19,9 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#ifndef SOFA_COMPONENT_MASS_DIAGONALMASS_H
-#define SOFA_COMPONENT_MASS_DIAGONALMASS_H
-#include "config.h"
-
+#pragma once
 
+#include "config.h"
```

Comment:
I would write 
`#include <SofaBaseMechanics/config.h>`
while you are at it 💁‍♂️